src/Database/Schema/PostgresSchema.php

```diff
@@ -112,7 +112,7 @@ protected function _convertColumn(string $column): array
             return ['type' => TableSchema::TYPE_UUID, 'length' => null];
         }
         if ($col === 'char' || $col === 'character') {
-            return ['type' => TableSchema::TYPE_STRING, 'fixed' => true, 'length' => $length];
+            return ['type' => TableSchema::TYPE_CHAR, 'fixed' => true, 'length' => $length];
```

Comment:
Why is the `fixed` key removed only from `MysqlSchema` and not from schema classes for other dbs?

Reply:
oversight! Thanks for pointing that...